### ISSUE TITLE
Migrate content from v5 "Queries, Search And Indexing"

### ISSUE THREAD

**Issue by stevepiercy** [plone/documentation#1730]
### Links of affected pages in Plone Documentation, if any.

- https://6.docs.plone.org/backend/search.html

### Description

Migrate content from Plone 5 docs.

- https://5.docs.plone.org/develop/plone/searching_and_indexing/index.html
- https://5.docs.plone.org/develop/plone/searching_and_indexing/catalog.html
- https://5.docs.plone.org/develop/plone/searching_and_indexing/query.html
- https://5.docs.plone.org/develop/plone/searching_and_indexing/external-catalogs.html
- https://5.docs.plone.org/develop/plone/searching_and_indexing/indexing.html
- https://5.docs.plone.org/external/plone.app.dexterity/docs/advanced/catalog-indexing-strategies.html

The last two need to be merged, as they contain duplicate information, but both have some information the other lacks.

The files are already converted to MyST on the `5.2-myst` branch.

https://github.com/plone/documentation/tree/5.2-myst/5.2-myst

**Comment by Manas-Kenge** [comment]
Do the Plone 5 docs just need to be copied into the Plone 6 docs, or do they need updates for the new version?

**Comment by stevepiercy** [comment]
They need a content expert to review their accuracy for Plone 6. That could mean overhauling, updating, or copying them. That's the situation for all the Plone 5 docs that have not yet been migrated to Plone 6.

**Comment by boss6825** [comment]
@stevepiercy  Can I do this by first reviewing and validating the existing Plone 5 docs, and then creating updated versions for Plone 6 based on them...
also, is there a central place where I can check the major changes from Plone 5 to 6 (for example, release notes or migration guides)?

**Comment by stevepiercy** [comment]
@boss6825 in addition to the resources in the issue's description, you can mine https://6.docs.plone.org/backend/upgrading/ for relevant changes.

I'd suggest working on only a section of a page at a time, especially if it's long and complicated. That will help it get reviewed faster, too. I'd be happy with small incremental baby steps, as that would be better than the current situation. Thank you!

**Comment by boss6825** [comment]
yeah its quite long as it involves indexing , catalogs ,  Querystring, security and Language, and much more.
 I think i should split it into 2-3 PRs as it is getting quite long....
